terraform/install.sh

```diff
@@ -165,7 +165,8 @@ applyTerraform() {
   rm -f .terraform/terraform.tfstate
 
   log "Initialize terraform backend with bucket ${bucket_name}"  
-  if [[ $(terraform init -backend-config "bucket=${bucket_name}" -lock=false) ]]; then
+  
+  if [[ $(terraform init -backend-config "bucket=${bucket_name}" -lock=false 2> /dev/null | cat) ]]; then
```

Comment:
Won't this always return true now?

Reply:
`If [[ ` statements expect some kind of comparitor, this line as is may give us unexpected results